templates/seL4SharedDataWithCaps.template.c

```diff
@@ -51,16 +51,15 @@
 /*- set shmem_symbol_size = "MAX_UNSAFE(%s, %s)" % (type_size, dataport_size) -*/
 /*? macros.shared_buffer_symbol(sym=dataport_symbol_name, shmem_size=shmem_symbol_size, page_size=page_size) ?*/
 
-/*- set arch_name = obj_space.spec.arch.capdl_name() -*/
-/*- if arch_name in ['aarch32', 'aarch64', 'arm_hyp', 'ia32', 'x86_64'] -*/
+/*- if options.architecture in ['aarch32', 'aarch64', 'ia32', 'x86_64'] -*/
```

Comment:
This is now missing arm_hyp

Reply:
Thanks, I had my test setup wrong and thus always saw `aarch32` there.